doc/source/docs/user_guide/set_operations.rst

```diff
@@ -135,60 +135,58 @@ but with the geometries obtained from overlaying ``df1`` with ``df2``:
     df2.plot(ax=ax, facecolor='none', edgecolor='k');
 
 
-Overlay countries example
+Overlay groceries example
 -------------------------
 
-First, load the countries and cities example datasets and select:
+First, load the Chicago boroughs and groceries example datasets and select :
```

Comment:
```suggestion
First, load the Chicago boroughs and groceries example datasets and select:
```
Also, just wondering if consistency matters, in mergingdata.rst the dataset is refered to as community areas rather than boroughs (which is also the terminology the metadata uses https://geodacenter.github.io/data-and-lab//comarea_vars/)